TLM/TLM/Util/RoundaboutMassEdit.cs

```diff
@@ -320,5 +350,25 @@ public class RoundaboutMassEdit {
             }
             return false;
         }
+
+        public void ClearNode(ushort nodeId) {
+            PriorityRoad.ClearNode(nodeId);
+            netService.IterateNodeSegments(nodeId, (ushort segmentId, ref NetSegment seg) => {
+                if (!HasJunctionFlag(nodeId)) {
+                    // clear stay in lane.
+                    LaneConnectionManager.Instance.RemoveLaneConnectionsFromNode(nodeId);
+                }
+                return true;
+            });
+        }
+
+        public void ClearRabout(List<ushort> segmentList) {
```

Comment:
Please prefer full name `ClearRoundabout`
/// <summary>Removes lane connectors from all nodes in this roundabout.</summary>